src/Components/Home/About/About.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "../../../scss/pages/_about.scss";
+
+function About() {
+  var Photo = "media/Asset 12@4x.png";
```

Comment:
These variables are not reused anywhere, I think it's better to directly use the paths